src/restaurant_inspections_api/services.clj

```diff
@@ -84,11 +84,12 @@
   (db/select-counties-summary))
 
 (defn- format-businesses-params
+  "Returns the parameters map with the zip codes as an array and page calculated"
   [params-map]
   (assoc params-map
-         :zipCodes (when-let [zipCodes (:zipCodes params-map)]
-                     (str/split zipCodes #","))
-         :page (* (:page params-map) (:perPage params-map))))
+         :zipCodes (when-let [zip-codes (:zipCodes params-map)]
```

Comment:
Yes, `when-let` serves as a guard for `str/split` otherwise, if we don't pass any zipcode, it will crash with `NullPointerException`